tests/utils/selectAtom.test.tsx

```diff
@@ -150,9 +146,11 @@ it('useSelector with scope', async () => {
   }
 
   const Selector = () => {
-    const a = useSelector(
-      bigAtom,
-      useCallback((value) => value.a, [])
+    const a = useAtomValue(
+      selectAtom(
+        bigAtom,
+        useCallback((value) => value.a, [])
+      )
```

Comment:
```suggestion
      useMemo(() => selectAtom(
        bigAtom,
        (value) => value.a
      ), [bigAtom])
```

Inside component, we always need useMemo to make the derived atom stable. Let's do so in tests too.